org.moneymanagerex.MMEX.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.moneymanagerex.MMEX
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
it would be better to build webkit 4.0 instead.

Reply:
@ovari Can upstream MMX be updated to build against webkit2gtk-4.1? Building webkit takes a long time, so it would be preferable to use whatever is provided by the SDK.

Reply:
as I said previously, it's wxWidgets the problem.

Reply:
Can you make a suggested change here, please?

Reply:
> Can upstream MMX be updated to build against webkit2gtk-4.1? Building webkit takes a long time, so it would be preferable to use whatever is provided by the SDK.

@vomikan can you please help?

Thank you

Reply:
Upstream wxwidgets will have to update its dependencies to handle a newer webkitgtk, but in the meantime it appears that building webkitgtk works with the newest Gnome SDK.